src/core/usubscription.rs

```diff
@@ -26,6 +26,41 @@ pub use crate::up_core_api::usubscription::{
 
 use crate::{UStatus, UUri};
 
+// Tracks information for the SubscriptionCache
```

Comment:
Use triple forward slashes for doc comments
```suggestion
/// Tracks information for the SubscriptionCache
```

Reply:
Would be good to expand this a bit with a code usage example a in triple tick block, e.g.
```rust
/// ```
/// // put the usage here
/// ```
```

that way it'll be compiled and tested.